Sources/Swift/Integrations/SessionReplay/SentryBreadcrumbReplayConverter.swift

```diff
@@ -0,0 +1,92 @@
+@_implementationOnly import _SentryPrivate
+import Foundation
+
+@objcMembers
+class SentryBreadcrumbReplayConverter: NSObject {
+    
+    private let supportedNetworkData = Set<String>([
+        "status_code",
+        "method",
+        "response_content_length",
+        "request_content_length",
+        "http.query",
+        "http.fragment"]
+    )
+    
+    func replayBreadcrumbs(_ breadcrumbs: [Breadcrumb], from: Date, until: Date) -> [SentryRRWebEvent] {
+        breadcrumbs.filter {
+            guard let timestamp = $0.timestamp else { return false }
+            return timestamp >= from && timestamp <= until
+        }
+        .compactMap { replayBreadcrumb(from: $0) }
+    }
+    
+    /**
+     * This function will convert the SDK breadcrumbs to session replay breadcrumbs in a format that the front-end understands.
+     * Any deviation in the information will cause the breadcrumb or the information itself to be discarded
+     * in order to avoid unknown behavior in the front-end.
+     */
+    private func replayBreadcrumb(from breadcrumb: Breadcrumb) -> SentryRRWebEvent? {
```

Comment:
For the RN SDK (and Flutter later), there should be a way to extend or override this so that it can implement mappings for its own breadcrumbs. AFAIU there's currently no way to instantiate `SentrySessionReplay` from a hybrid SDK so it can't provide its own implementation. Or has anything changed in that regard lately? It came up in the discussion in [this issue](https://github.com/getsentry/sentry-cocoa/issues/2616) but I haven't noticed any updates.

Reply:
Nothing new yet. Still need to find a proper way to make SentrySessionReplay Hybrid public

Reply:
Alright, with breadcrumbs this starts to be a blocker for react-native too so when do you think you could have a look into that?

Reply:
I guess @bruno-garcia can help us with the priorities. 
Right now Im working with touch tracker. 
I can work on exposing SessionReplay for hybrids after it, or I can give it a pause right now.

Reply:
I'd say it's OK to push multi touch in order to unblock a react native breadcrumbs. We need to get that one out